invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js

```diff
@@ -121,6 +122,14 @@ class QueryBuilder {
     return this;
   }
 
+  withStartDate({ fromDate, toDate }) {
```

Comment:
I would name it `withStartDateRange`, because `withStartDate` looks for me like we only take one date

Reply:
@kprzerwa so has to be then also the updated. I can see your point, but as I read the function name `with...` I see the last part of it being the property name that affects the query somehow and not the way we filter on it. We can of course change it!!

Reply:
I would rename to something like withDateRange. WDYT?